Simplenote/Simplenote-Info.plist

```diff
@@ -55,6 +53,8 @@
 			</dict>
 		</array>
 	</dict>
+	<key>ITSAppUsesNonExemptEncryption</key>
+	<false/>
```

Comment:
Nice, one less thing to update manually in AppStoreConnect's UI when submitting 👍